app/controllers/keyword_type.js

```diff
@@ -0,0 +1,82 @@
+"use strict";
+
+const KeywordType = require('../models/keyword_type');
+
+/**
+ * Get all KeywordTypes in the database
+ * @param  {Express.Request}   req  - the request object
+ * @param  {Express.Response}  res  - the response object
+ * @param  {Function} next - pass to next route handler
+ */
+function getKeywordTypes(req, res, next) {
+    return KeywordType.fetchAll()
+    .then(collection => res.json(collection.serialize()))
+    .catch(e =>console.error(e))
```

Comment:
Good thinking catching the error, but this isn't how we want to handle it. This would be caused by a database error, so we would want to respond with a 500. All this does now is just silently log the error, then time out.